js/main.js

```diff
@@ -1,2 +1,41 @@
 /* global data */
 /* exported data */
+
+var urlInput = document.querySelector('#input-url');
+var photo = document.querySelector('#main-image');
+
+function updatePhoto(event) {
+  photo.setAttribute('src', event.target.value);
+  if (event.target.value === '') {
+    photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  }
+}
+
+urlInput.addEventListener('input', updatePhoto);
+
+var form = document.querySelector('#form');
+var title = document.querySelector('#title-text');
+var notes = document.querySelector('#notes-text');
+var savedJournal = [];
+var entryID = 0;
```

Comment:
There should be no need for these variables, you should just be using your data model stored in the `data` variable in `data.js`